conftest.py

```diff
@@ -1,9 +1,14 @@
 """Global configuration for pytest"""
 
+import os
 import numpy as np
 import pytest
 
 
+# System fonts can interfere with screenshots that expect a particular font to be used
+os.environ["PYGFX_DISABLE_SYSTEM_FONTS"] = "1"
```

Comment:
Headsup @hmaarrfk, I set this env var in `conftest.py`. It was in `pytest.ini`, but from what I can tell, this never worked unless you have `pytest-env` installed.

Reply:
ah you are correct. sorry about that. I guess that is pulled in in my dev envs.

I understand why you would go this route instead of adding pytest-env.

I forgot what the motivation was for me using that pytest-env variable. It might have been to switch between Qt backends in my QtGUI applicaiton. Sometimes code runs at import time, so you can't be sure hte line you added gets set in time.